compatibility-changer/compatibility-changer/requirements.txt

```diff
@@ -0,0 +1,6 @@
+certifi==2024.8.30
```

Comment:
do we need all of them explicitly? should we omit transitive dependencies and let pip figure it out?